econml/sklearn_extensions/ensemble.py

```diff
@@ -555,8 +552,7 @@ def fit(self, X, y, sample_weight=None, sample_var=None):
                                                                        int(np.ceil(self.subsample_fr_ *
                                                                                    (X.shape[0] // 2))),
                                                                        replace=False)])
-            res = Parallel(n_jobs=self.n_jobs, verbose=self.verbose,
-                           **_joblib_parallel_args(prefer='threads'))(
+            res = Parallel(n_jobs=self.n_jobs, verbose=self.verbose, prefer='threads')(
```

Comment:
Similarly here, can we verify that _joblib_parallel_args only returns prefer='threads' and not some other keyword too.

Reply:
this is handled in the joblib versions we require